src/Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -213,6 +213,18 @@
                         <FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="RunWithPowerShellButton"
+                    Command="{x:Bind ViewModel.RunWithPowerShellCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay}"
```

Comment:
Is this needed if the button is already hidden?

Reply:
No not needed, I can remove it.